python/paddle/trainer_config_helpers/layers.py

```diff
@@ -4972,7 +4976,16 @@ def nce_layer(input,
     """
     if isinstance(input, LayerOutput):
         input = [input]
+        assert not isinstance(param_attr, collections.Sequence)
+        param_attr = [param_attr]
+    else:
```

Comment:
please add simple usage in annotation when input is not LayerOutput, i.e, add simple usage for else branch.